internal/plugins/helm/v1/chartutil/chart.go

```diff
@@ -152,7 +152,7 @@ func CreateChart(projectDir string, opts CreateOptions) (*resource.Options, *cha
 		return nil, nil, fmt.Errorf("failed to load chart: %v", err)
 	}
 
-	log.Infof("Created %s", relChartPath)
+	fmt.Printf("Created %s\n", relChartPath)
```

Comment:
Why replace the log by fmt?

Reply:
To align with other command print statements.

Reply:
It is strange we use fmt instead of the log lib.